R/03-geojob-obj.R

```diff
@@ -56,6 +56,31 @@ setMethod("geojob", signature(), function(...) {
   return(geojob)
 })
 
+#'@param xml location of xml 
+#'@rdname geojob-methods
+#'@importFrom httr GET 
+#'@importFrom RCurl url.exists
+#'@importFrom XML xmlTreeParse xmlAttrs xmlRoot
+#'@aliases geojob,geojob-method
+setMethod("geojob", signature("ANY"), function(xml, urlSlot = NULL) {
```

Comment:
With S4 objects, the method signature allows you to have a specific function for the class of the input. Using "ANY" means it will try to do the same thing on any input to this function for the `xml` parameter input. I think writing a method for `signature="XMLAbstractDocument"` might be a good starting point, similar to #332 

Also, S4 methods can't use defaults for the arguments, so the `urlSlot = NULL` is going to be ignored and you'll get an error along the lines of `urlSlot is missing and...`